README.md

```diff
@@ -59,10 +52,13 @@ export SQL_PORT=<PORT>
 
     pip install paho-mqtt
 
-### Create a user
+## Create a user
 
     ./manage.py createsuperuser
 
-# View / Edit Notebooks
+## Adding requirements
+
+To add a requirement, add it with no version constraint (or as little as needed)
+to `requirements.in` (or `requirements-dev.in` or `requirements-prod.in` if it is needed only in prod or dev). Then run `pip-compile` (or `pip-compile requirements-dev.in` or `pip-compile requirements-prod.in`).
 
-    ./manage.py shell_plus --notebook
+Never edit a `requirements-*.txt` file by hand !
```

Comment:
Maybe worth to add a word about dependabot here :wink:

Reply:
```suggestion
Never edit a `requirements-*.txt` file by hand ! (dependabot is allowed to do it though)
```

Reply:
Huhu, I had something more explanatory in mind, as not everyone might be familiar with Dependabot :stuck_out_tongue: 

```suggestion
Never edit a `requirements-*.txt` file by hand !

In addition, we use [Dependabot](https://dependabot.com/) who will automatically submit Pull Requests to upgrade the python packages when a new version is available. This will only change the `requirement-*.txt`.
```